cohttp-lwt/src/client.mli

```diff
@@ -4,4 +4,4 @@
     module.  The resulting module satisfies the {! Client } module type. *)
 
 module Make (IO:S.IO) (Net:S.Net with module IO = IO) : S.Client
-  with type ctx = Net.ctx
+  with type resolvers = Net.resolvers
```

Comment:
This renaming causes interface churn -- could we just keep the `ctx` name in this release?  Most people will just pass it through unmodified, but the type renaming will break libraries that use the Cohttp functor.

Reply:
I'm ok with that :+1:.